src/nerve/server/handlers/workflow_handler.py

```
@@ -0,0 +1,300 @@
+            answer: User's answer (required)
+
+        Returns:
+            success: True if answered
+        """
+        session = self.session_registry.get_session(params.get("session_id"))
+        run_id = self.validation.require_param(params, "run_id")
+        answer = self.validation.require_param(params, "answer")
+
+        run = session.get_workflow_run(run_id)
+        if run is None:
+            raise ValueError(f"Workflow run '{run_id}' not found in session '{session.name}'")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)